tests/test_io.py

```diff
@@ -241,6 +241,22 @@ def test_sdf_props_and_conformer_preserved(tmp_path):
     np.testing.assert_almost_equal(conf.GetPositions(), pos, decimal=4)
 
 
+def test_read_mol2(datadir):
+    data_path = datadir / "test.mol2"
+
+    # to list of mols
+    mols = dm.read_mol2file(data_path)
+
+    for mol in mols:
+        assert isinstance(mol, Chem.rdchem.Mol)
```

Comment:
Minor but you can also use `dm.Mol`.

Reply:
![Screen Shot 2023-03-30 at 6 16 19 PM](https://user-images.githubusercontent.com/69532598/228976010-48d09f45-f106-4633-8af5-61713e93ce92.png)

Hmm, out of consistency I tried to emulate your codebase as much as possible.
So the image above showing Chem.rdchem.Mol
Should I keep it as `Chem.rdchem.Mol` or `dm.Mol`?

Reply:
Good point and probably a legacy from the past xD

It does not really matter, feel free to change it or not.

Reply:
Regardless, I kept as is!